core/src/test/scala/cromwell/util/SampleWdl.scala

```diff
@@ -520,41 +521,7 @@ object SampleWdl {
 
   object ArrayIO extends SampleWdl {
     override def wdlSource(runtime: String = "") =
-      s"""task concat_files {
```

Comment:
The serialize task remains so that it can be used for checking file identification. The rest is now in centaur so all filesystems can benefit from its regression testing